README.md

```diff
@@ -12,7 +12,7 @@ There are two ways to run `kn quickstart`:
 
 1. You can run it standalone, just put it on your system path and make sure it is executable.
 2. You can install it as a plugin of the `kn` client to run:
-    * Follow the [documentation] to install `kn client` if you don't have it
+    * Follow the [documentation](https://github.com/knative/client/blob/main/docs/README.md#installing-kn) to install `kn client` if you don't have it
```

Comment:
just to check, did we copy this readme from anywhere (template repo etc) that we should also update? (Seems like it might be something that's shared)

Reply:
I borrowed the text from one of the other plugin repos (not a template though), but the missing link was just me forgetting to put it in when I was writing out the readme